src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -6395,244 +6399,272 @@ namespace OutputReportTabular {
 								} else {
 									tableBody( columnRecount - 1, lMonth ) = "-";
 									tableBody( columnRecount, lMonth ) = "-";
 								}
+							} //lMonth
+							// add the summary to bottom
+							// Don't include if the original min and max values are still present
+							if ( minVal < veryLarge ) {
+								tableBody( columnRecount - 1, 15 ) = RealToStr( minVal, digitsShown );
 							} else {
-								tableBody( columnRecount - 1, lMonth ) = "-";
-								tableBody( columnRecount, lMonth ) = "-";
+								tableBody( columnRecount - 1, 15 ) = "-";
 							}
-						} //lMonth
-						// add the summary to bottom
-						// Don't include if the original min and max values are still present
-						if ( minVal < veryLarge ) {
-							tableBody( columnRecount - 1, 15 ) = RealToStr( minVal, digitsShown );
-						} else {
-							tableBody( columnRecount - 1, 15 ) = "-";
-						}
-						if ( maxVal > verySmall ) {
-							tableBody( columnRecount - 1, 16 ) = RealToStr( maxVal, digitsShown );
-						} else {
-							tableBody( columnRecount - 1, 15 ) = "-";
-						}
-					}}
-				} //KColumn
-				WriteReportHeaders( MonthlyInput( iInput ).name, MonthlyTables( curTable ).keyValue, isAverage );
-				WriteSubtitle( "Custom Monthly Report" );
-				WriteTable( tableBody, rowHead, columnHead, columnWidth, true ); //transpose monthly XML tables.
-				if ( sqlite ) {
-					sqlite->createSQLiteTabularDataRecords( tableBody, rowHead, columnHead, MonthlyInput( iInput ).name, MonthlyTables( curTable ).keyValue, "Custom Monthly Report" );
-				}
+							if ( maxVal > verySmall ) {
+								tableBody( columnRecount - 1, 16 ) = RealToStr( maxVal, digitsShown );
+							} else {
+								tableBody( columnRecount - 1, 15 ) = "-";
+							}
+						}}
+					} //KColumn
+					WriteReportHeaders( MonthlyInput( iInput ).name, MonthlyTables( curTable ).keyValue, isAverage );
+					WriteSubtitle( "Custom Monthly Report" );
+					WriteTable( tableBody, rowHead, columnHead, columnWidth, true ); //transpose monthly XML tables.
+					if ( sqlite ) {
+						sqlite->createSQLiteTabularDataRecords( tableBody, rowHead, columnHead, MonthlyInput( iInput ).name, MonthlyTables( curTable ).keyValue, "Custom Monthly Report" );
+					}
+				} // if ( missingHourAggError || missingMaxOrMinError )
 			} //jTables
+			if ( missingMaxOrMinError ) {
+				ShowWarningError( "The Output:Table:Monthly report named=\"" + MonthlyInput( iInput ).name + "\" has a valueWhenMaxMin aggregation type for a column without a previous column that uses either the minimum or maximum aggregation types. The report will not be generated." );
+			}
+			if ( missingHourAggError ) {
+				ShowWarningError( "The Output:Table:Monthly report named=\"" + MonthlyInput( iInput ).name + "\" has a --DuringHoursShown aggregation type for a column without a previous field that uses one of the Hour-- aggregation types. The report will not be generated." );
+			}
```

Comment:
If we aren't generating the report, is it worth just shutting it down with a fatal here?  I'd hate for a user to run an hour long simulation and quietly not get the report they were expecting.  Seems severe enough here to call fatal to me, but I don't feel _that_ strongly about it.